src/mainboard/emulation/qemu-riscv/src/print.rs

```diff
@@ -0,0 +1,21 @@
+use core::fmt;
```

Comment:
do we need to propagate this file to every new target? can't it go $TOP/src/lib/print.rs?

Reply:
Yes it can. I'll do that in the next version